ui/src/components/HomePage/PopularGridSection.jsx

```
@@ -1,37 +1,42 @@
-              <CardTitle className="text-lg">{p.name}</CardTitle>
-              <p className="text-sm text-gray-500">{p.brand}</p>
-              <p className="font-semibold mt-2">${p.price}</p>
-            </CardContent>
-          </Card>
-        ))}
-      </div>
-    </section>
-  );
+    <div className='flex max-w-7xl *:rounded-none *:shadow-none max-xl:flex-col max-xl:*:not-last:border-b-0 max-xl:*:first:rounded-t-xl max-xl:*:last:rounded-b-xl xl:*:not-last:border-r-0 xl:*:first:rounded-l-xl xl:*:last:rounded-r-xl'>
+      {products.map((prod) => (
+        <Card className='overflow-hidden pt-0 w-full'>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)